lib/k8sClient.js

```diff
@@ -2,6 +2,8 @@
 
 const k8s = require('k8s'),
     debug = require('debug')('helmit:k8sClient');
+var STRICT_SSL = process.env.STRICT_SSL || 'doc,iso';
```

Comment:
do we need this? Don't see it being used anywhere. To get over not having the correct cert on the hosts we need to set `process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";` At that is what I have found out to be true. What is this variable for?